fusor-ember-cli/app/utils/foreman-task-util.js

```diff
@@ -0,0 +1,30 @@
+// ForemanTaskUtil: Wrappers around driving foreman tasks
```

Comment:
I was about to ask for some documentation about the class, then I saw this. :+1: